File: Abubakr Khanooni/Arrays/richestCustomerWealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maxi=0;
        int sum;
        for(int i=0;i<accounts.size();i++){
            sum=0;
            for(int j=0;j<accounts[0].size();j++){
                sum+=accounts[i][j];
            }
            maxi=max(maxi,sum);
        }
        return maxi;
    }

};

File: Deepak Urs/Arrays/richestCustomerWealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int mx = -1;
        int rows = accounts.length;
        int cols = accounts[0].length;

        for(int r = 0; r < rows; r++) {
            int tmp = 0;
            for(int c = 0; c < cols; c++) {
                tmp += accounts[r][c];
            }
            mx = Math.max(mx, tmp);
        }

        return mx;
    }
}